### ISSUE TITLE
vibe.d and external threads

### ISSUE THREAD

**Issue by John-Colvin** [rejectedsoftware/vibe.d#1445]
```D
import vibe.d;
import std.stdio;

void main()
{
    import std.concurrency : spawn;
    spawn({ while(1){} });

    runTask({ writeln("hello"); });

    runEventLoop();
}
```

never prints `hello`.

**Comment by s-ludwig** [comment]
`spawn` currently spawns a *task* instead of a thread when using vibe.d. Maybe it should rather run a worker task, though, since the interface is already assuming a separate thread anyway. I don't really remember the reason for the current choice.

**Comment by s-ludwig** [comment]
Was fixed by f5468032 - uses `runWorkerTask` by default and can also be configured to spawn a new thread instead.
